From 15b7efa2d0b5c2f75542d5d91c2b914c3131cbbe Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Fri, 9 Sep 2022 16:42:07 +0200
Subject: [PATCH 75/84] arm64: dts: rockchip: rk3399-pinephone-pro: Add
 magnetometer sensor support

Pinephone Pro uses AF8133J.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
(cherry picked from commit 6aa4fc189489549c065a1474b42f52ab3ca9f034)
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index 2a2568caf975..927b481bd588 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -971,6 +971,14 @@ usbc_ss: endpoint {
 			};
 		};
 	};
+
+	af8133j: compass@1c {
+		compatible = "voltafield,af8133j";
+		reg = <0x1c>;
+		reset-gpios = <&gpio1 RK_PA1 GPIO_ACTIVE_LOW>;
+		avdd-supply = <&vcc_3v0>;
+		dvdd-supply = <&vcc_1v8>;
+	};
 };
 
 &i2s0 {
-- 
2.55.0

